testsuite/forge/src/backend/k8s/chaos/network_delay.yaml

```diff
@@ -20,7 +20,7 @@ spec:
   # to (the packets to target)
   # both ( the packets from or to target)
   # This parameter makes Chaos only take effect for a specific direction of packets.
```

Comment:
nit: edit this comment :p

Reply:
I believe this is generic line is generic enough. It only talks about possible values right?

Reply:
oh nevermind, ignore lol I'm misreading my own past comment